src/features/dialog-manager/dialogs/character-creation/index.js

```diff
@@ -7,15 +7,26 @@ import Dialog from '../../../../components/dialog';
 import createCharacter from '../../actions/create-character';
 import setClass from '../../actions/set-class';
 import setRace from '../../actions/set-race';
+import errorMessage from '../../actions/error-message';
 
 import './styles.scss';
 
-const CharacterCreation = ({ dialog, createCharacter, setClass, setRace }) => {
+const CharacterCreation = ({
+    dialog,
+    createCharacter,
+    errorMessage,
+    setClass,
+    setRace,
+}) => {
     function handleContinue() {
         dialog.character.characterName = document.getElementById(
             'characterName'
         ).value;
-        createCharacter();
+        if (dialog.character.characterName) {
+            createCharacter();
```

Comment:
Do we want them to only insert certain characters here? It's not an  issue for this, because it works, but I can enter '     ' and it'll still work (I'm not even sure it's necessarily an issue, and probably shouldn't block this PR but I just wanted a quick discussion around it).

Reply:
I think there should be a check for whitespace (so that entering tabs/spaces doesn't count), but as long as it doesn't explicitly _break_ anything, I'm fine with someone entering a name like 文部科学省検定済教科書, ¯\_(ツ)_/¯ or حسان.